src/services/bookService.ts

```diff
@@ -34,3 +35,14 @@ export const getAllBooks = async (): Promise<Books[]> => {
         connection.execSql(request);
     });
 };
+
+export const getAllBooksBySequelize = async () => {
+    try {
+        await sequelize.authenticate();
+        console.log('Connection');
+    } catch (error) {
+        console.error('No connection');
+    }
```

Comment:
Will this be done in all functions that make db queries? Might be worth extracting.